### PR TITLE
chore(website): add RSS feed

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

This PR adds a simple RSS feed to the website. The only issue I found it that we are not able to actually put the whole content of a blog post due to some malformed HTML. 

As alternative, I put the description inside the content

## Test Plan

The rendered website should create a new `/feed.xml` page

### CODE REVIEW

**Comment by MichaReiser** [comment]
What's the use case for the RSS feed? What's our goal?

**Comment by ematipico** [comment]
> What's the use case for the RSS feed? What's our goal?

On twitter some user asked us if the blog has an RSS/Atom feed, because they would like to subscribe to it. I don't see any reason to say no, and the plugin is very simple to integrate.

**Review by MichaReiser — APPROVED** [review]

File: website/.eleventy.js [hunk 1]
```diff
@@ -198,6 +200,10 @@ module.exports = function (eleventyConfig) {
 		return string.toLowerCase().replace(/\s/g, "-");
 	});
 
+	eleventyConfig.addFilter("withAbsoluteUrl", function (string) {
+		return "https://rome.tools" + string;
```
Comment (MichaReiser) [inline website/.eleventy.js:204]: Is it safe to manually concatenated the string passed here? Or should we use some URL module?

Comment (ematipico) [reply website/.eleventy.js:204]: I think so, the string coming from the filter is the permalink specified in the markdown of the blog post

**Pull request merged**